### ISSUE TITLE
Still having odd display issue

### ISSUE THREAD

**Issue by MichaelTurk** [hilios/jQuery.countdown#100]
After implementing the CSS fix from issue #72, we're still seeing odd display flickers and artifacts. To make sure it was not just our instance, I pulled up the countdown homepage and was able to duplicate the problem there (see attached).

Sometimes it is a relatively small line like this:

![screen-shot-2015-02-18-at-11 20 44-am](https://cloud.githubusercontent.com/assets/7374226/6251485/2e5dbae6-b762-11e4-99ad-780127627df8.jpg)

Sometimes it is significantly larger:

![countdown-oddity](https://cloud.githubusercontent.com/assets/7374226/6251493/37ae697e-b762-11e4-9c46-56355dd7cfc8.jpg)
 
Anyone have thoughts as to what would be causing that? We're seeing it on a number of displays, cards, and OSes, so it doesn't seem to be specific to the hardware or the OS. It does, however, seem to be more common in Chrome than any other browser.

**Comment by hilios** [comment]
Thank you for your feedback, but this problem seems to be caused by hardware acceleration, dunno if there is a way around by css/html.

I never saw this bug on my computer though, and I use Chrome has my main browser, I think we need to investigate the real cause of this issue.

**Comment by dingo-d** [comment]
This happens on Chrome and Opera if I recall correctly. On FF there is no such bug, so it's definitely a rendering issue.

**Comment by MichaelTurk** [comment]
Is it specific to the "flip" clock implementation? I have used the clock
without the flip animation and never had issues. So I'm wondering if it's
something in that specific use case.

**Comment by dingo-d** [comment]
For me it happened only with the flip clock. From what I can tell other people only have the issue with the flip clock as well.

**Comment by hilios** [comment]
Guys, it's an issue with the CSS of the flip clock, I really don't have much experience with 3d animations and hardware acceleration at CSS level. Do you guys can guess where we begin to solve this problem?

**Comment by vilius-g** [comment]
The issue seems to go away when the perspective value for `.main-example .time` is under `480px`. When I set it to `479px` everything is displayed perfectly, but when I increase it to `480px` the artifacts come back. This is very weird but seems to work for me.

**Comment by dingo-d** [comment]
For me it goes away when I put perspective to 0px;

**Comment by hilios** [comment]
The problem is not the script itself, but the machine hardware
acceleration! The output will dependent on your computer

Em sex, 27 de mar de 2015 06:04, dingo-d <notifications@github.com>
escreveu:

**Comment by dingo-d** [comment]
Yeah, I think it also depends on the browser, because if I recall correctly, on firefox the issue didn't exist. And ff is known to handle css3 transformations better than chrome (or webkit based browsers).

**Comment by vilius-g** [comment]
Another thing I noticed that at `transform: rotateX(90deg)` the card is still visible. It seems that the transformation origin is not aligned properly.
